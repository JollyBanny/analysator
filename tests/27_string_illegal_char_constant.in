'Hello, World!'
'I think this legal string literal'
'Hello, Worl'##33