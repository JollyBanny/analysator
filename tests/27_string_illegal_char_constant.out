1	1	String	Hello, World!	'Hello, World!'
2	1	String	I think this legal string literal	'I think this legal string literal'
3	1	Illegal char constant
